CHANGES.md

```diff
@@ -90,6 +90,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 ### Fixed and improved
 
+* Backends for requests to `/service/<service-name>` may compress responses (DCOS_OSS-4906)
```

Comment:
Note to self, maybe for an editorial change later on:

Admin Router's service endpoint now forwards the `Accept-Encoding` header if present in the request. This for example allows for gzip-compressed responses.